### PR TITLE
Add type rules for vectorfields as symbols

### PR DESCRIPTION
**Pull request by lukem12345**

### CODE REVIEW

File: src/acset.jl [hunk 1]
```diff
@@ -427,12 +430,12 @@ function safe_modifytype!(d::SummationDecapode, var_idx::Int, new_type::Symbol)
 end
 
 """
-    filterfor_forms(types::AbstractVector{Symbol})
+    filterfor_ec_types(types::AbstractVector{Symbol})
```
Comment (jpfairbanks) [inline src/acset.jl:433]: should we have both? if we rename, have to bump minor version number.

File: test/language.jl [hunk 2]
```diff
@@ -831,6 +832,38 @@ end
...
+
+      M == ♯(I)
+      N == ♯(J)
+      O == ♭(K)
+      P == ♭(L)
+    end
+    infer_types!(d)
+
+    # TODO: Update this as more sharps and flats are released.
+    names_types_expected = Set([(:A, :Form1), (:B, :DualForm1), (:C, :PVF), (:D, :DVF),
```
Comment (jpfairbanks) [inline test/language.jl:861]: This should be a Dict{Symbol, Symbol} right? each variable needs a unique type.

**Review by jpfairbanks — CHANGES_REQUESTED** [review]

File: test/language.jl (hunk 2 above)
Comment (lukem12345) [reply test/language.jl:861]: That would rely on the property of names in a Decapode being unique

File: src/acset.jl (hunk 1 above)
Comment (lukem12345) [reply src/acset.jl:433]: This function is not exported

File: test/language.jl (hunk 2 above)
Comment (jpfairbanks) [reply test/language.jl:861]: Good point.

**Review by jpfairbanks — APPROVED** [review]

**Pull request merged**